### PR TITLE
build: Add support for shield specific code. Zephyr 3.2

### PR DESCRIPTION
**Pull request by Nicell**
Updated #11 to work with Zephyr 3.2. Works with the new nice!view PR at https://github.com/zmkfirmware/zmk/pull/1768

### CODE REVIEW

File: cmake/modules/shields.cmake [hunk 1]
```diff
@@ -71,6 +71,10 @@ foreach(root ${BOARD_ROOT})
         continue()
       endif()
 
+      if(BOARD_DIR AND NOT (${root} STREQUAL ${ZEPHYR_BASE}))
```
Comment (petejohanson) [inline cmake/modules/shields.cmake:74]: Some funky indentation here.

**Review by petejohanson — CHANGES_REQUESTED** [review]

File: cmake/modules/shields.cmake (hunk 1 above)
Comment (Nicell) [reply cmake/modules/shields.cmake:74]: Looks fine on my side. It's indented to the for loop nest level and then the inside of the if block is indented properly. Am I missing something obvious here? 🤔

Comment (petejohanson) [reply cmake/modules/shields.cmake:74]: Nope...GH mobile rendering things poorly. All good actually.

**Review by petejohanson — APPROVED** [review]
Thanks!

**Pull request merged**